docs/conf.py

```diff
@@ -49,9 +49,12 @@
 # intersphinx mapping
 intersphinx_mapping = {
     'numpy': ('https://numpy.org/doc/stable/', None),
+    'numpydoc': ('https://numpydoc.readthedocs.io/en/latest/', None),
```

Comment:
Avoid linking to the dev version
```suggestion
    'numpydoc': ('https://numpydoc.readthedocs.io/en/stable/', None),
```